File: arch/src/main/java/com/github/lany192/arch/tab/TabItem.java
package com.github.lany192.arch.tab;

import androidx.fragment.app.Fragment;

public class TabItem {
    private final CharSequence title;
    private final Fragment fragment;

    public TabItem(CharSequence title, Fragment fragment) {
        this.title = title;
        this.fragment = fragment;
    }

    public CharSequence getTitle() {
        return title;
    }

    public Fragment getFragment() {
        return fragment;
    }
}

File: library/src/main/java/com/github/lany192/box/adapter/TabItem.kt
package com.github.lany192.box.adapter

import androidx.fragment.app.Fragment

class TabItem {
    var title: String
    var fragment: Fragment

    constructor(title: String, fragment: Fragment) {
        this.title = title
        this.fragment = fragment
    }

    constructor(fragment: Fragment) {
        title = fragment.javaClass.simpleName
        this.fragment = fragment
    }
}